src/config-schema.ts

```diff
@@ -113,6 +113,13 @@ export const configSchema = {
       'Enable or disable stock deduction during the dispensing process. Requires the stock management module to be installed and configured.',
     _default: false,
   },
+
+  validateBatch: {
+    _type: Type.Boolean,
+    _description:
+      'Enable or disable stock item batch number validation.Requires the stock management module to be installed and configured.',
+    _default: false,
```

Comment:
This seems like `_default: true` is the right way of keeping the current behaviour by default?